### PR TITLE
fix name of vector_id column

### PR DESCRIPTION
**Pull request by ttt-77**

### CODE REVIEW

File: tests/tasti_test/tasti_test.py [hunk 1]
```diff
@@ -128,14 +130,14 @@ def test(
...
-         embedding_dim=128, nb_buckets=1000, index_path='./')
-
-    # too slow
-    print(f'Running Weaviate vector database')
-    url = ''
-    api_key = os.environ.get('WEAVIATE_API_KEY')
-    weaviate_auth = WeaviateAuth(url, api_key=api_key)
-    test('Weaviate', VectorDatabaseType.WEAVIATE.value, data_size=200,
-         embedding_dim=128, nb_buckets=50, weaviate_auth=weaviate_auth)
+    # print(f'Running Chroma vector database')
```
Comment (ddkang) [inline tests/tasti_test/tasti_test.py:133]: What's going on here?

Comment (ttt-77) [reply tests/tasti_test/tasti_test.py]: Fixed it.

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
 
     - name: Download test data
       run: |
-        gdown https://drive.google.com/uc?id=1SyHRaJNvVa7V08mw-4_Vqj7tCynRRA3x
+        gdown https://drive.google.com/uc?id=1a0EA7sH_m0m9c1127Geq_bNnDmi7nj3x
```
Comment (ddkang) [inline .github/workflows/tests.yaml:24]: Why was this changed?

Comment (ttt-77) [reply .github/workflows/tests.yaml:24]: Previous blob mapping table csv file's column is vector_id, I changed it to __vector_id. And I also added a law dataset.

**Review by ddkang — APPROVED** [review]
